### PR TITLE
Maximize workbench shell & make sure its visible when tests are started

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

**Comment by rawagner** [comment]
testPR

**Comment by mlabuda** [comment]
Could you please add tests with a custom shell? E.g. create a new shell, perform operations on it and check whether it is in expected state.

**Comment by mlabuda** [comment]
testPR

File: plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/before/test/impl/MaximizeAndFocusWorkbench.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+/******************************************************************************* 
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at http://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation 
+ ******************************************************************************/
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/before/test/impl/MaximizeAndFocusWorkbench.java]: please move license header at the top of the file.

**Pull request merged**